elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -115,6 +116,11 @@ def attach_cve_flaws_cli(runtime, advisory_id, noop, default_advisory_type):
         ))
         advisory.update(security_impact=highest_impact)
 
+    if highest_impact not in advisory.topic:
```

Comment:
Line 117 only updates the security impact when the current impact of advisory is smaller than `highest_impact`. I think the topic should be updated together with advisory security impact change. Any thought?

Reply:
Agreed, it's better to always set to the highest_impact. I removed the if statement here.